Modules/IO/TransformMINC/CMakeLists.txt

```diff
@@ -2,11 +2,8 @@ project(ITKIOTransformMINC)
 set(ITKIOTransformMINC_LIBRARIES ITKIOTransformMINC)
 
 if(NOT ITK_USE_SYSTEM_HDF5)
-  include_directories(
-    "${ITKHDF5_SOURCE_DIR}/src/itkhdf5"
-    "${ITKHDF5_SOURCE_DIR}/src/itkhdf5/src"
-    "${ITKHDF5_BINARY_DIR}/src/itkhdf5"
-    )
+  include_directories("${ITKHDF5_SOURCE_DIR}/src/itkhdf5" "${ITKHDF5_SOURCE_DIR}/src/itkhdf5/src"
```

Comment:
This was more readable previously. Could we enforce one dir per line?

Reply:
I think we can make one argument per line more relevant per configuration.